src/System.Net.Primitives/tests/FunctionalTests/System.Net.Primitives.Functional.Tests.csproj

```diff
@@ -33,13 +33,14 @@
     <Compile Include="IPAddressMappingTest.cs" />
     <Compile Include="IPEndPointTest.cs" />
     <Compile Include="NetworkCredentialTest.cs" />
-    <Compile Include="RequestCachePolicyTest.cs" Condition="'$(TargetGroup)' == ''" />
     <Compile Include="SocketAddressTest.cs" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetGroup)' == ''">
     <Compile Include="LoggingTest.cs" />
     <Compile Include="SerializationTest.cs" />
+    <Compile Include="RequestCachePolicyTest.cs" />
     <Compile Include="CookieContainerAddTest.cs" />
+    <Compile Include="NetworkCredentialTest.netstandard1.7.cs" />
```

Comment:
@safern Thanks for explaining! I'm starting to understand why this case is special. 

@weshaggard @ericstj /cc @karelz @davidsh to confirm: What I think is that AOT/UWP is still based on ns1.3 and S.N.Primitives is still somehow getting tested against that profile. Other contracts such as S.N.Security were not part of the profile and so are not seeing this issue. Is the name `.netstandard1.7.cs` correct (since we're calling it netstandard2.0)?